TestAssets/TestProjects/PerformanceTestProjects/TwoTargetGraphLarge/TwoTargetLargeP6/project.json

```diff
@@ -1,13 +1,19 @@
 {
   "version": "1.0.0-*",
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "type": "platform",
-      "version": "1.0.0-rc3-004324"
-    }
-  },
+  "dependencies": { },
   "frameworks": {
-    "netcoreapp1.0": {},
-    "netstandard1.5": {}
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-004336"
+        }
+      }
+    },
+    "netstandard1.5": {
```

Comment:
Oh I see. `Microsoft.NETCore.App` was fixed to no longer target `netstandard`. This makes sense now.